bin/crew

```diff
@@ -1271,10 +1271,14 @@ end
 def install_files(src, dst = src.delete_prefix('.'))
   if File.executable?("#{CREW_PREFIX}/bin/crew-mvdir") && !CREW_DISABLE_MVDIR
     system "crew-mvdir #{@short_verbose} #{src} #{dst}"
-  elsif File.executable?("#{CREW_PREFIX}/bin/rsync")
-    system "rsync -ah#{@verbose}HAXW --remove-source-files #{src}/ #{dst}"
   else
-    system "cd #{src}; tar -cf - ./* | (cd #{dst}; tar -x#{@verbose}p --keep-directory-symlink -f -)"
+    warn 'crew-mvdir is not installed. Please install it with \'crew install crew_mvdir\''.yellow
```

Comment:
@uberhacker Is this reasonable or do you think there needs to be more explanation?

Reply:
You could mention that installing crew-mvdir can help improve performance?

Reply:
Maybe this would work:
```
warn 'crew-mvdir is not installed. Please install it with \'crew install crew_mvdir\' for increased installation performance'.yellow
```